src/main/java/ch/njol/skript/expressions/ExprConsumedItem.java

```diff
@@ -0,0 +1,65 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.EventRestrictedSyntax;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityShootBowEvent;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Consumed Item")
```

Comment:
I think this should maybe mention the event.
Due to minecraft having the consumable component, even tho Skript doesn't have it... yet...
This name is a bit misleading.

Reply:
The descriptive explicitly states "the item consumed within an entity shoot bow event", I don't believe there's any reason to further add more. The naming was decided to be generic so other events can more easily fit their values in here when needed.

Reply:
Ahh yah ok thats fair.